hack/scripts/patch-remote/patch_remote.py

```diff
@@ -130,7 +137,7 @@ def _resolve_targets(self, _targets):
             if _targets
             else self._config.get("PATCH_TARGETS", ["dashboard"])
         )
-        if len(targets) == 0:
+        if not targets or len(targets) == 0:
```

Comment:
you can leave it to be `If not targets:` because in python it's the same:
```
>>> if not list:
...     print("empty")
...
empty
>>> list=None
>>> if not list:
...     print("empty")
...
empty
```